scaleway/resource_k8s_pool.go

```diff
@@ -239,6 +238,16 @@ func resourceScalewayK8SPoolCreate(ctx context.Context, d *schema.ResourceData,
 		KubeletArgs: expandKubeletArgs(d.Get("kubelet_args")),
 	}
 
+	if v, ok := d.GetOk("region"); ok {
+		req.Region = scw.Region(v.(string))
+	}
+
+	if v, ok := d.GetOk("zone"); ok {
```

Comment:
This zone is related to the control plane. This will not work for the multicloud option where the resource can be in any zone. In order to handle this behavior, we should get use the `placement_group_id` with their zone.